CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 
 ## v1.6.0
 2020-08-??
+### Upgrade requirements
+- If you are  running Layman with development or test settings, set [LAYMAN_SETTINGS_MODULE](doc/env-settings.md#LAYMAN_SETTINGS_MODULE) in your `.env` file to `layman_settings`. This is not needed if you are running Layman with demo settings.
```

Comment:
What about production?

Reply:
As far as I know, all current production instances use demo settings. So no action is required.

Anyway, it's good point to mention it. I am going to rephrase it.